database/migrations/2019_07_12_045144_create_plans_table.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreatePlansTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        if (!Schema::connection('dbp_users')->hasTable('plans')) {
+            Schema::connection('dbp_users')->create('plans', function (Blueprint $table) {
+                $table->bigIncrements('id');
+                $table->string('name');
+                $table->boolean('featured')->default(false);
+                $table->integer('user_id')->unsigned();
+                $table->date('suggested_start_date');
+                $table->foreign('user_id', 'FK_plans')->references('id')->on(config('database.connections.dbp_users.database') . '.users')->onDelete('cascade')->onUpdate('cascade');
+                $table->softDeletes();
+                $table->timestamp('created_at')->default(DB::raw('CURRENT_TIMESTAMP'));
+                $table->timestamp('updated_at')->default(DB::raw('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'));
+            });
+        }
+
+        if (!Schema::connection('dbp_users')->hasTable('plan_days')) {
+            Schema::connection('dbp_users')->create('plan_days', function (Blueprint $table) {
+                $table->bigIncrements('id');
+                $table->bigInteger('plan_id')->unsigned();
+                $table->foreign('plan_id', 'FK_plan_days')->references('id')->on(config('database.connections.dbp_users.database') . '.plans')->onDelete('cascade')->onUpdate('cascade');
+                $table->timestamp('created_at')->default(DB::raw('CURRENT_TIMESTAMP'));
+                $table->timestamp('updated_at')->default(DB::raw('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'));
+                $table->integer('order_column');
+            });
+        }
+
+        if (!Schema::connection('dbp_users')->hasTable('plan_playlist_items')) {
```

Comment:
plan days and playlists are so similar, i suggest they be stored in the same table.  in which case, this table name should be playlist_items.  however, if there are good reasons to store them separately, this table could be renamed play_day_items (to avoid confusion with playlists)

Reply:
hmm, has FK to playlist_id, which isn't in this PR so I'll need to go find that to understand the relationship here

Reply:
Done Jon, now the name of the table is `playlist_items` and each `plan_day` have now a playlist